src/XRP/model/xrp-transaction.ts

```diff
@@ -176,6 +190,7 @@ export default class XRPTransaction {
   private constructor(
     readonly hash: string,
     readonly account?: string,
+    readonly accountXAddress?: string,
```

Comment:
Curious on your thoughts: We have `account` and `sourceTag`. Should we also make a sourceXAddress? 

I think yes.

Reply:
derived this internally too